libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Input/ConfirmInput.cs

```diff
@@ -87,6 +87,17 @@ public ConfirmInput([CallerFilePath] string callerPath = "", [CallerLineNumber]
         [JsonProperty("outputFormat")]
         public ValueExpression OutputFormat { get; set; }
 
+        /// <inheritdoc/>
+        public override RecognizerDescription GetRecognizerDescription(DialogContext dialogContext, string expectedLocale)
+            => new RecognizerDescription(entities: new[] { new EntityDescription("boolean") });
```

Comment:
>new EntityDescription("boolean") [](http://example.com/codeflow?start=59&length=32)

Do we need to capture the forms of boolean?  Yes/NO/OK/Cancel/Nope, etc.?

Reply:
No.  This corresponds to the boolean prebuilt in luis or the confirmation in speech.